### PR TITLE
feat(aiven_mirrormaker_replication_flow): add config_properties_exclude

### PR DESCRIPTION
**Pull request by rriski**
* Add `config_properties_exclude` attribute to `aiven_mirrormaker_replication_flow` resource
* Test `topic_blacklist` attribute in `aiven_mirrormaker_replication_flow` tests

### CODE REVIEW

File: internal/sdkprovider/service/kafka/mirrormaker_replication_flow.go [hunk 1]
```diff
@@ -102,6 +103,15 @@ var aivenMirrorMakerReplicationFlowSchema = map[string]*schema.Schema{
 		Description:  "Offset syncs topic location. Possible values are `source` & `target`. There is no default value.",
 		ValidateFunc: validation.StringInSlice([]string{"source", "target"}, false),
 	},
+	"config_properties_exclude": {
```
Comment (keejon) [inline internal/sdkprovider/service/kafka/mirrormaker_replication_flow.go:106]: Hm only issue i might see is that we have max 2048 chars string here in [API](https://github.com/aiven/aiven-core/blob/15e1e5933c0d568433d884e21baf6268d79835e4/aiven/web/schema.py#L3660). Could potentially be confusing if it fails due to this but i don't think a major issue.

**Review by keejon — DISMISSED** [review]
LGTM 👍

**Review by ssuomi-aiven — APPROVED** [review]

File: docs/data-sources/mirrormaker_replication_flow.md [hunk 2]
```diff
@@ -33,6 +33,7 @@ data "aiven_mirrormaker_replication_flow" "f1" {
 
 ### Read-Only
 
+- `config_properties_exclude` (List of String) List of topic configuration properties and/or regular expressions to not replicate. The properties that are not replicated by default are: `follower.replication.throttled.replicas`, `leader.replication.throttled.replicas`, `message.timestamp.difference.max.ms`, `message.timestamp.type`, `unclean.leader.election.enable`, and `min.insync.replicas`. Setting this overrides the defaults. For example, to exclude only 'min.insync.replicas' and 'unclean.leader.election.enable'  set this to: ["follower\\\\.replication\\\\.throttled\\\\.replicas", "leader\\\\.replication\\\\.throttled\\\\.replicas", "message\\\\.timestamp\\\\.difference\\\\.max\\\\.ms",  "message\\\\.timestamp\\\\.type"]
```
Comment (keejon) [inline docs/data-sources/mirrormaker_replication_flow.md:36]: a little confusing to me. "to exclude only 'min.insync.replicas' and 'unclean.leader.election.enable" refers to exclude it from the defaults?

Comment (rriski) [reply docs/data-sources/mirrormaker_replication_flow.md:36]: FYI @staceysalamon-aiven. Yes, that refers to excluding the properties from the defaults.

Comment (rriski) [reply docs/data-sources/mirrormaker_replication_flow.md]: Updated to "For example, to enable replication for..."

**Review by ssuomi-aiven — APPROVED** [review]

**Pull request merged**